terraform/setup/vpc.tf

```diff
@@ -32,6 +32,19 @@ resource "aws_security_group" "ec2_security_group" {
     cidr_blocks = ["0.0.0.0/0"]
   }
 
+  egress {
```

Comment:
could you annotate these please

Reply:
like how andrew did for the others

Reply:
To be simple, the [extra apps](https://github.com/aws/amazon-cloudwatch-agent-test/blob/main/terraform/ecs_fargate/linux/default_resources/default_extra_apps.tpl#LL1) will expose two ports: 6379 and 9121 and its will follow the prometheus protocol. Afterwards, CWA will scrape the prometheus metrics with those 2 apps. Therefore, 6379 an 9121  needs to be exposed in order to scraped. For more information on for the full flow, you can [follow this documentation](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/ContainerInsights-Prometheus-Setup-redis-ecs.html). However, since the security group are being shared between [extra apps](https://github.com/aws/amazon-cloudwatch-agent-test/blob/main/terraform/ecs_fargate/linux/main.tf#L129) and[ the cwagent](https://github.com/aws/amazon-cloudwatch-agent-test/blob/main/terraform/ecs_fargate/linux/main.tf#L89), therefore, extra apps needs to expose the 6379 and 9121 (egress) while cwagent needs able to access those ports (ingress)